### DISCUSSION TITLE
With new UI, remove port auto-increment on duplicate env

### DISCUSSION THREAD

**Discussion by Gogolian** [mockoon/mockoon#604] [Ideas] [answered]
Hello and upfront THANKS for doing a great job! mockoon is awesome!

Yet however awesome it may be i think it seems logical now that while previously port was easily visible right now, when hidden under Settings tab, it MAY NOT be a great idea to auto-increment port while duplicating environments.

1. It may confuse new users - they would have to dig around to find why the port has automatically changed while they would expect an exact copy of configuration.
2. For users like me, who use mockoon to switch between some configurations, but consumer app is always expecting same port i have to manually switch it after each environment duplication

An alternative may be a checkbox in the global options to disable auto-increment / keep port on duplication.

Many thanks for hearing out! Keep doing a great job!

**Comment by 255kb** [comment, marked as answer]
Thanks @Gogolian for the kind words 😊
I thought the port auto-increment was a nice addition. But I guess you are right, it may be confusing as it disappeared from the main view. 
I will create an issue to track this and see if there is any interest. Before removing anything or adding another option, I would like to see if this opinion is shared or not.
